omrsigcompat/omrsig.cpp

```diff
@@ -276,6 +277,34 @@ sigaction(int signum, const struct sigaction *act, struct sigaction *oldact) __T
 }
 #endif /* defined(POSIX_SIGNAL) */
 
+#if defined(WIN32)
+HMODULE
```

Comment:
Can we not figure out at runtime what the current MSVC runtime module is?

Reply:
When omrsig uses a different version of the MSVCR than the application, the whole thing is borked anyways. If we want, I could determine the version of MSVCR in cmake, and we can load that specific version here. That would essentially move this check to compile-time. Sound okay?